src/util/icon/styles.js

```diff
@@ -33,21 +33,24 @@ export const createIconStyleSet = ( attrNameTemplate = '%s', mainClassName = '',
 
 	const styles = []
 
+	const clampedTabletSize = clampValue( getValue( 'Size' ), { max: 200 } )
+	const clampedMobileSize = clampValue( getValue( 'Size' ), { max: 200 } )
+
 	styles.push( {
 		[ `.${ mainClassName } .ugb-icon-inner-svg, .${ mainClassName } .ugb-icon-inner-svg svg` ]: {
 			width: appendImportant( getValue( 'Size', '%spx' ) ),
 			height: appendImportant( getValue( 'Size', '%spx' ) ),
 		},
-		tablet: {
+		tabletOnly: {
 			[ `.${ mainClassName } .ugb-icon-inner-svg, .${ mainClassName } .ugb-icon-inner-svg svg` ]: {
-				width: appendImportant( getValue( 'TabletSize', '%spx' ) ),
-				height: appendImportant( getValue( 'TabletSize', '%spx' ) ),
+				width: appendImportant( getValue( 'TabletSize', '%spx' ) || `${ clampedTabletSize }px` ),
+				height: appendImportant( getValue( 'TabletSize', '%spx' ) || `${ clampedMobileSize }px` ),
```

Comment:
It's possible for this to become `undefinedpx` if `clampedTabletSize` is `undefined`